synthtool/gcp/templates/python_notebooks_testing_pipeline/.cloud-build/execute_changed_notebooks.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
```

Comment:
This file also looks out-of-date, it should match https://github.com/GoogleCloudPlatform/ai-platform-samples/blob/master/.cloud-build/ExecuteChangedNotebooks.py

Reply:
Will do - I think it's just out of date because the PR has been in flight for a minute.